app/src/ui/changes/changes-list.tsx

```diff
@@ -560,13 +560,38 @@ export class ChangesList extends React.Component<
       { type: 'separator' },
     ]
     if (paths.length === 1) {
+      const shouldEnableContextMenu = Path.basename(path) !== GitIgnoreFileName
```

Comment:
Just a nit, not required, but if you named this variable `enabled` then the following menu items could be reduced from `enabled: shouldEnableContextMenu` to `enabled,`